embed/etcd.go

```diff
@@ -257,19 +257,21 @@ func startClientListeners(cfg *Config) (sctxs map[string]*serveCtx, err error) {
 		}
 
 		proto := "tcp"
+		addr := u.Host
 		if u.Scheme == "unix" || u.Scheme == "unixs" {
 			proto = "unix"
+			addr = u.Host + u.Path
```

Comment:
https://github.com/coreos/etcd/pull/7646/files#diff-d7a47eb75475dba263540eb7f8456e50R308 should be `sctxs[addr] = sctx`?